src/wui/sound_options.cc

```diff
@@ -17,6 +17,7 @@
  */
 
 #include "wui/sound_options.h"
+#include <graphic/text_layout.h>
 
 #include "base/i18n.h"
 #include "sound/sound_handler.h"
```

Comment:
```suggestion
#include "wui/sound_options.h"

#include "base/i18n.h"
#include "graphic/text_layout.h"
#include "sound/sound_handler.h"
```
Always use `""` instead of `<>` for Widelands includes. We really need a codecheck rule for this ;)